modules/apps/27-interchain-accounts/controller/types/msgs.go

```diff
@@ -12,6 +12,8 @@ import (
 	ibcerrors "github.com/cosmos/ibc-go/v8/modules/core/errors"
 )
 
+const MaximumOwnerLength = 2048 // maximum length of the owner in bytes
```

Comment:
should we document somewhere that this limit is arbitrarily selected?

Reply:
I will add this in the feature branch in all the places where we have limits.